Command/CheckTranslationProgressCommand.php

```diff
@@ -3,22 +3,30 @@
 namespace OpenClassrooms\Bundle\OneSkyBundle\Command;
 
 use OpenClassrooms\Bundle\OneSkyBundle\Model\Language;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use OpenClassrooms\Bundle\OneSkyBundle\Services\LanguageService;
 use Symfony\Component\Console\Helper\Table;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
 
 /**
  * @author Romain Kuzniak <romain.kuzniak@openclassrooms.com>
  */
-class CheckTranslationProgressCommand extends ContainerAwareCommand
+class CheckTranslationProgressCommand extends Command
 {
-    const COMMAND_NAME = 'openclassrooms:one-sky:check-translation-progress';
+    public const COMMAND_NAME = 'openclassrooms:one-sky:check-translation-progress';
 
-    const COMMAND_DESCRIPTION = 'Check translations progress';
+    public const COMMAND_DESCRIPTION = 'Check translations progress';
```

Comment:
do these constants need to be public? are they accessed somewhere outside of `getCommandName` and `getCommandDescription`?

Reply:
Yup, they're used in the tests